Orange/widgets/visualize/owsilhouetteplot.py

```diff
@@ -494,6 +494,7 @@ def setScores(self, scores, labels, values, colors, rownames=None):
         self.__setup()
 
     def setRowNames(self, names):
+        ROW_NAMES_WIDTH = 200
```

Comment:
Introducing a constant is OK since you explain what `200` is. But having it here is almost as if it just above the line where it's used. What about moving this constant to module- or class-level (https://www.python.org/dev/peps/pep-0008/#constants)?